files/run_httpd.sh

```diff
@@ -1,19 +1,31 @@
 #!/usr/bin/bash
 
-set -xe
+set -eux
 
-source /usr/bin/setup_env_in_openshift.sh
-
-pushd /src
 # if all containers started at the same time, pg is definitely not ready to serve
 # so let's try this for a few times
+ATTEMPTS=7
 n=0
-until [ $n -ge 7 ]
-do
+while [[ $n -lt $ATTEMPTS ]]; do
   alembic-3 upgrade head && break
   n=$[$n+1]
   sleep 2
 done
-popd  # pushd /src
 
-httpd -DFOREGROUND
+# If the number of attempts was exhausted: the migration failed.
+# Exit with an error.
+if [[ $n -eq $ATTEMPTS ]]; then
+    exit 1
+fi
```

Comment:
Can we echo that the migration fails so it's visible in the logs?

Reply:
Added an error message.